common-arcana.lic

```diff
@@ -73,14 +73,13 @@ module DRCA
   end
 
   def start_barb_abilities(abilities, settings)
-    # Note, you must know Power meditation or Powermonger mastery
-    # for your active abilities to be detected by DRSpells.
-    abilities
-      .reject { |name| DRSpells.active_spells[name] }
-      .each { |name| activate_barb_buff?(name, settings.meditation_pause_timer) }
+    abilities.each { |name| activate_barb_buff?(name, settings.meditation_pause_timer) }
   end
 
   def activate_barb_buff?(name, meditation_pause_timer = 20)
+    # Note, you must know Power meditation or Powermonger mastery
+    # for your active abilities to be detected by DRSpells.
+    return true if DRSpells.active_spells[name]
```

Comment:
Not sure why this check was originally in `start_barb_abilities` instead of this function. Makes sense to me to put the logic in `activate_barb_buff?` so that whatever invokes the method can take advantage of this optimization.